package.json

```diff
@@ -41,6 +41,7 @@
     "helmet": "^3.15.0",
     "memory-cache": "^0.2.0",
     "morgan": "^1.9.1",
+    "pg": "^7.12.0",
```

Comment:
For some reason it complained about this so I added the dep

Reply:
That's the driver for Postgres needed with Sequelize